### PR TITLE
667 yw measure overlappingInDegreeAv

### PR DESCRIPTION
**Pull request by maverickyuwei**
testing the draft request
- [x] Implement OverlappingInDegreeAv.m
- [x] Implement test_OverlappingInDegreeAv.m

### CODE REVIEW

File: braph2/graph/measures/OverlappingInDegreeAv.m [hunk 1]
```diff
@@ -0,0 +1,154 @@
+classdef OverlappingInDegreeAv < OverlappingInDegree
+    % OverlappingInDegreeAv Average overlapping in-degree measure
+    % OverlappingInDegreeAv provides the average overlapping in-degree of a graph
+    % for binary directed (BD) and weighted directed (WD) multiplexes. 
+    %
+    % It is calculated as the average of the sum of the number of inward edges 
+    % connected to the node in all layers, i.e., it is the sum of the 
+    % in-degree of a node in all layers.
```
Comment (annacanal) [inline braph2/graph/measures/OverlappingInDegreeAv.m:8]: it is the **average** of the sum of the in-degree

File: braph2/graph/measures/OverlappingInDegreeAv.m [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    %   getParametricity            - returns the parametricity of the measure 
+    %   getMeasure                  - returns the average overlapping in-degree class
+    %   getCompatibleGraphList      - returns a list of compatible graphs
+    %   getCompatibleGraphNumber    - returns the number of compatible graphs
+    %
+    % See also Measure, OverlappingInDegree, InDegree, MultiplexGraphBD, MultiplexGraphWD.
+    
+    methods
+        function m = OverlappingInDegreeAv(g, varargin)
+            % OVERLAPPINGINDEGREE(G) creates average overlapping in-degree with default properties.
```
Comment (annacanal) [inline braph2/graph/measures/OverlappingInDegreeAv.m:29]: OVERLAPPINGINDEGREEAV(G)

File: braph2/graph/measures/OverlappingInDegreeAv.m [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+            % See also Measure, OverlappingInDegree, InDegree, MultiplexGraphBD, MultiplexGraphWD.
+            
+            m = m@OverlappingInDegree(g, varargin{:});
+        end
+    end
+    methods (Access=protected)
+        function overlapping_in_degree_av = calculate(m)
+            % CALCULATE calculates the average overlapping in-degree value of a multiplex
+            %
+            % OVERLAPPINGin-degree = CALCULATE(M) returns the value of the average overlapping
```
Comment (annacanal) [inline braph2/graph/measures/OverlappingInDegreeAv.m:42]: OVERLAPPINGINDEGREEAV = CALCULATE(M)

File: braph2/graph/measures/OverlappingInDegreeAv.m [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+            % in-degree of a multiplex.
+            %
+            % See also Measure, Overlappingin-degree, in-degree, MultiplexGraphBU, MultiplexGraphWU.
```
Comment (annacanal) [inline braph2/graph/measures/OverlappingInDegreeAv.m:45]: Use the name of the class:
OverlappingInDegree, InDegree, MultiplexGraphBU

File: braph2/graph/measures/OverlappingInDegreeAv.m [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+                'The average overlapping in-degree of a network is the average ' ...
+                'of the sum of the inward edges connected to the node in all layers. ' ...
+                'Connection weights are ignored in calculations.' ...
+                ];
+        end
+        function available_settings = getAvailableSettings()
+            % GETAVAILABLESETTINGS returns the setting available to OverlappingInDegreeAv
+            %
+            % AVAILABLESETTINGS = GETAVAILABLESETTINGS() returns the
+            % settings available to OverlappingInDegree. Empty Array in this case.
```
Comment (annacanal) [inline braph2/graph/measures/OverlappingInDegreeAv.m:95]: OverlappingInDegreeAv

**Review by annacanal — CHANGES_REQUESTED** [review]
Small things on the documentation. Check the proposed changes on the comments
For the measure itself everything good ^^

File: braph2/graph/measures/OverlappingInDegreeAv.m (hunk 1 above)
Comment (maverickyuwei) [reply braph2/graph/measures/OverlappingInDegreeAv.m:8]: thank you anna!

File: braph2/graph/measures/OverlappingInDegreeAv.m (hunk 3 above)
Comment (maverickyuwei) [reply braph2/graph/measures/OverlappingInDegreeAv.m:42]: thank you anna!

File: braph2/graph/measures/OverlappingInDegreeAv.m (hunk 4 above)
Comment (maverickyuwei) [reply braph2/graph/measures/OverlappingInDegreeAv.m:45]: thank you anna!

File: braph2/graph/measures/OverlappingInDegreeAv.m (hunk 5 above)
Comment (maverickyuwei) [reply braph2/graph/measures/OverlappingInDegreeAv.m:95]: thank you anna!

File: braph2/graph/measures/OverlappingInDegreeAv.m (hunk 2 above)
Comment (maverickyuwei) [reply braph2/graph/measures/OverlappingInDegreeAv.m]: thank you anna!

**Review by annacanal — APPROVED** [review]

**Pull request merged**